src/components/checkout/address-form-summary.js

```diff
@@ -1,33 +1,60 @@
 // Libraries
-import React from 'react'
+import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 
 // Lib
 import componentMapping from '../../lib/component-mapping'
 
-function AddressFormSummary ({ addressLine1, city, firstName, lastName, onClick, postcode, showEditButton }) {
-  const AddressFormHeader = componentMapping('AddressFormHeader')
-
-  return (
-    <>
-      <AddressFormHeader
-        collapsed
-        onClick={onClick}
-        title='Shipping Address'
-        showEditButton={showEditButton}
-      />
+class AddressFormSummary extends PureComponent {
+  constructor (props) {
+    super(props)
+
+    this.AddressFormHeader = componentMapping('AddressFormHeader')
+  }
+
+  renderAddress () {
+    const { 
+      addressLine1,
+      city,
+      firstName,
+      lastName,
+      postcode
+    } = this.props
+
+    return(
       <div className='o-form__wrapper--collapsed c-address-form__summary'>
-        <p className='u-bold'>{ firstName } { lastName } </p>
-        <span>{ addressLine1 }, { city }, { postcode }</span>
+        <p className='u-bold'>{firstName} {lastName} </p>
+        <span>{addressLine1}, {city}, {postcode}</span>
       </div>
-    </>
-  )
+    )
+  }
+
+  render () {
+    const {
+      headerTitle,
+      onClick,
+      showEditButton
+    } = this.props
+
+    return (
+      <>
```

Comment:
going forward we are going to use `<Fragment>` instead of using shorthand.
I would suggest changing any `<></>` to use `<Fragment></Fragment>`